packages/team-management/src/constants.tsx

```diff
@@ -0,0 +1,17 @@
+export const API_BASE_URL = 'https://opensrp-stage.smartregister.org/opensrp/rest/';
```

Comment:
We should try have this defined in the client env, incase we need to deploy this work in another server instance.

Reply:
@HaseemUlHaq  ?